tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -25,6 +25,9 @@ ulimit -a
 # - try adding a dependency under a language-specific section first (reduces latency and increases build stability)
 # - only add stuff that you absolutely need for your builds to work (add comment to explain why its needed)
 
+# Disable HOMEBREW update to avoid new updates which potentially have problems.
+export HOMEBREW_NO_AUTO_UPDATE=1
```

Comment:
sg.
suggestion: would it be cleaner to also explictly checkout homebrew-core so that things are at a known state?
Like this we are relying on what is the last commit available on kokoro macos workers and we have no control of that.  I think it would be worth checking what's the commit provided by the current mojave kokoro workers and pinning it (or at least mentioning it in a comment, so that troubleshooting is easier).

Reply:
I briefly updated it to include more info on why its update is disabled and the current version of the core repository. I didn't fix the version because it's meant to use the version when the image was built and it will change when migrating to Catalina or Big sure in future.